src/connectedk8s/HISTORY.rst

```diff
@@ -3,6 +3,10 @@
 Release History
 ===============
 
+1.5.5
+++++++
+* Setting guard default authnMode to 'arc' when RBAC is enabled.
```

Comment:
Please be more specific here. `guard` is implementation details.

Reply:
You can describe all the changes we made such as the deprecation of the flags etc...